monetio/sat/_tropomi_l2_no2_mm.py

```diff
@@ -0,0 +1,138 @@
+# Reading TROPOMI L2 NO2 data
+
+import logging
+import os
+import sys
+from collections import OrderedDict
+from glob import glob
+
+import numpy as np
+import xarray as xr
+from netCDF4 import Dataset
+
+# from monetio.hdf.hdfio import hdf_open, hdf_close, hdf_list, hdf_read
+
+
+def read_dataset(fname, variable_dict):
+    """
+    Parameters
+    __________
```

Comment:
```suggestion
    ----------
```
And the other ones in this module too.